index.html

```diff
@@ -12,8 +12,107 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
+</head>
```

Comment:
Hmm. This looks weird... it's almost as if someone forgot to put in their meta viewport tag... 🤔

![image](https://user-images.githubusercontent.com/8473052/57823265-d675f380-7764-11e9-8657-826260b468af.png)